bletl/growth.py

```diff
@@ -325,16 +357,16 @@ def fit_mu_t(
         # Override guess with user-provided mu_prior for nonzero starting points.
         mu_guess[mu_prior != 0] = mu_prior[mu_prior != 0]
 
-    # build PyMC3 model
+    # build PyMC model
     coords = {
         "timepoint": numpy.arange(TD),
         "segment": numpy.arange(TS),
     }
-    with pymc3.Model(coords=coords) as pmodel:
-        pymc3.Data('known_switchpoints', t_switchpoints_known)
-        pymc3.Data('t_data', t_data, dims="timepoint")
-        pymc3.Data('t_segments', t_segments, dims="segment")
-        dt = pymc3.Data('dt', numpy.diff(t_data), dims="segment")
+    with pm.Model(coords=coords) as pmodel:
+        pmData('known_switchpoints', t_switchpoints_known)
+        pmData('t_data', t_data, dims="timepoint")
+        pmData('t_segments', t_segments, dims="segment")
+        dt = pmData('dt', numpy.diff(t_data), dims="segment")
```

Comment:
```suggestion
        dt = pm.Data('dt', numpy.diff(t_data), dims="segment")
```

Reply:
The `pmData` without the dot is on purpose. In the import line 15/17 it aliases the new `pm.ConstantData` or the old `pm.Data` depending on the PyMC version.